canonical-web-and-design/greenhouse-exporter
# Remove csv file dependency

## Done
- Now data in `app.py` is fetched from the original database so there's no need to query our database (and thus  `hiring_leads.csv`)
## QA
- VPN
- `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- Go to http://0.0.0.0:8501/
- Check the job details table is working
